luci-app-wrtbwmon/htdocs/luci-static/resources/view/wrtbwmon/details.js

```diff
@@ -76,28 +150,20 @@ function parseDatabase(values, hosts) {
 	return cachedData;
 }
 
-function parseSize(size){
-	var unit = ['' , 'K', 'M', 'G', 'T', 'P', 'E', 'Z'];
-	var num = parseFloat(size);
-	var base = (size).match(/[KMGTPEZ]/i);
-	var ex = unit.indexOf(base ? (base).toString().toUpperCase() : '');
-
-	return Math.round((num || 1) * (ex != -1 ? 1024 ** ex : 1));
-}
-
-function progressbar(query, v, m, byte) {
+function progressbar(query, v, m) {
 	var pg = $(query),
-	    vn = parseInt(v) || 0,
-	    mn = parseInt(m) || 100,
-	    fv = byte ? String.format('%1024.2mB', v) : v,
+	    vn = v || 0,
+	    mn = m || 100,
+	    fv = formatSpeedNative(v),
 	    pc = ((100 / mn) * vn).toFixed(2),
 	    wt = Math.floor(pc > 100 ? 100 : pc),
-	    bgc = (pc >= 95 ? 'red' : (pc >= 80 ? 'magenta' : (pc >= 60 ? 'yellow' : 'lime')));
-
+	    bgc = (pc >= 95 ? 'red' : (pc >= 80 ? 'darkorange' : (pc >= 60 ? 'yellow' : 'lime')));
+		var tc = (pc >= 80 ? 'white' : '#404040');
```

Comment:
The indentation seems incorrect.
You can write as
```suggestion
	    bgc = (pc >= 95 ? 'red' : (pc >= 80 ? 'darkorange' : (pc >= 60 ? 'yellow' : 'lime'))),
	    tc = (pc >= 80 ? 'white' : '#404040');
```